runtime/dancebox/src/lib.rs

```diff
@@ -529,6 +616,7 @@ impl SessionManager<AccountId> for CollatorsFromInvulnerablesAndThenFromStaking
 parameter_types! {
     pub const Period: u32 = prod_or_fast!(1 * HOURS, 1 * MINUTES);
     pub const Offset: u32 = 0;
+    pub const CollatorRotationSessionPeriod: u32 = 5;
```

Comment:
I would put it to something higher for now, like 12 or 24

Reply:
or probably we can configure this with prod_or_fast macro

Reply:
So what should be the final value?